mujoco_warp/_src/sensor.py

```diff
@@ -1950,7 +1952,8 @@ def _sensor_tactile(
 
   # compute distance
   plugin_id = geom_plugin_index[geom]
-  depth = wp.min(sdf(int(GeomType.SDF.value), lpos, plugin_attr[plugin_id], plugin[plugin_id]), 0.0)
+  volume_data = make_volume_data(wp.uint64(0), wp.vec3(0.0), wp.vec3(0.0))
```

Comment:
Can you please explain this? Will the sensor return the correct value when colliding with a Volume SDF? Will this slow down the sensor when colliding with a primitive SDF?